platform/v8.js

```diff
@@ -122,9 +122,8 @@ async function v8 (args) {
 
 // Public method so it can be used in external error handlers
 v8.getIsolateLog = function (workingDir, pid) {
-  return fs.readdirSync(workingDir).find(function (f) {
-    return new RegExp(`isolate-0(x)?([0-9A-Fa-f]{2,16})-${pid}-v8.log`).test(f)
-  })
+  const regex = new RegExp(`isolate-0(x)?([0-9A-Fa-f]{2,16})-${pid}-v8.log`)
+  return fs.readdirSync(workingDir).find(regex.test.bind(regex))
```

Comment:
Is the `.bind()` necessary at all?

Reply:
It just means we can pass the method directly without a wrapper function. 

`return fs.readdirSync(workingDir).find(regex.test)` hits an error because regex.test looks for the regex in `this`.

Alternative would be like `return fs.readdirSync(workingDir).find(function (f) { regex.test(f) })` referencing regex and calling its method inside a wrapper function.